addons/viewdistance/XEH_postInit.sqf

```diff
@@ -2,26 +2,28 @@
 
 if (!hasInterface) exitWith {};
 
+// Force the view distance down to the limit.
 if (viewDistance > GVAR(limitViewDistance)) then {
-    setViewDistance GVAR(limitViewDistance); // force the view distance down to the limit.
-    setObjectViewDistance (0.8 * GVAR(limitViewDistance));
-} else {
-    [true] call FUNC(adaptViewDistance); // adapt view distance in the beginning according to whether client is on foot or vehicle.
+    setViewDistance GVAR(limitViewDistance);
```

Comment:
You could use this to handle JIP:
```
["PlayerJip",{
    [false] call FUNC(adaptViewDistance)
}] call ace_common_fnc_addEventHandler;
```

Setting viewDistance to the limit might not be good because it does not respect client side settings.